### DISCUSSION TITLE
zellij integration: i want hx to automatically start in an zellij session

### DISCUSSION THREAD

**Discussion by wiiznokes** [helix-editor/helix#10382] [Q&A]
How can i do this please ?

I tried `echo hx | zellij` without sucess. And `zellij --help` doesn't help much.

**Comment by wiiznokes** [comment]
A workarround could be to write the output of `zellij setup --generate-auto-start bash` in `.bashrc`.

**Comment by danillos** [comment]
If you don't have zellij running you'll need https://zellij.dev/documentation/layouts

Something like `zellij --layout /path/to/your_helix_layout.kdl`

If you have zellij running you can Zellij CLI https://zellij.dev/documentation/controlling-zellij-through-cli
